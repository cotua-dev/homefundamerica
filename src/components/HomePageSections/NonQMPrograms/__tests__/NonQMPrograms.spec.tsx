import React from 'react';
import { render, RenderResult } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { default as NonQMPrograms } from '..';

describe('NonQMPrograms component', (): void => {
    test('renders', (): void => {
        const { container }: RenderResult = render(<NonQMPrograms/>);

        expect(container.firstChild).toMatchSnapshot();
    });
});
