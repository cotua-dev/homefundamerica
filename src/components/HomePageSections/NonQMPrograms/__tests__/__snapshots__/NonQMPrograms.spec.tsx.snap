// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NonQMPrograms component renders 1`] = `
<section>
  <h1>
    Non-QM Programs
  </h1>
</section>
`;
